pkgs/development/libraries/vigra/default.nix

```diff
@@ -20,28 +19,25 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "vigra";
-  version = "unstable-2022-01-11";
+  version = "1.12.1";
 
   src = fetchFromGitHub {
     owner = "ukoethe";
     repo = "vigra";
-    rev = "093d57d15c8c237adf1704d96daa6393158ce299";
-    sha256 = "sha256-pFANoT00Wkh1/Dyd2x75IVTfyaoVA7S86tafUSr29Og=";
+    rev = "Version-1-12-1";
```

Comment:
```suggestion
    tag = "Version-${lib.replaceStrings ["."] ["-"] version}";
```

Reply:
Yes, this is actually adressed in #375389